source/retryable-writes/tests/README.rst

```diff
@@ -395,10 +395,67 @@ and sharded clusters.
 
    9. Disable the failpoint.
 
+#. Test that drivers return the original error after encountering a
+   WriteConcernError error with a RetryableWriteError label. This test MUST be
+   implemented by any driver that implements the Command Monitoring
+   specification. This prose test requires drivers to set a fail point after an
+   ``insertOne`` operation but before the subsequent retry. Drivers that are
+   unable to set a failCommand after the CommandSucceededEvent SHOULD use
+   mocking or write a unit test to cover the same sequence of events.
+
+   1. Create a client with ``retryableWrites=true``. If testing against a
+      sharded deployment, the test runner MUST ensure that the client connects
+      to only a single mongos host.
+
+   2. Configure a fail point with error code ``91`` (ShutdownInProgress)::
+
+         db.adminCommand({
```

Comment:
[nit] It looks like there's an extra space at the beginning of the `db.adminCommand({` lines both here and on line 428. (9 spaces instead of 8 spaces.)